app/controllers/admins_controller.rb

```diff
@@ -7,5 +7,14 @@ class AdminsController < ApplicationController
 
   def show
     authorize! :admin_dashboard
+    @presenter = build_presenter
+
+    @presenter.work_stats = StatBuilder.build_stats if user_with_groups.administrator?
```

Comment:
Is this conditional necessary?  Aren't all users of this route administrators?

Reply:
That's true, got it.